avocado/plugins/multiplex.py

```diff
@@ -48,7 +48,7 @@ def configure(self, parser):
 
         parser.add_argument('--filter-out', nargs='*', default=[],
                             help='Filter out path(s) from multiplexing')
-        parser.add_argument('-s', '--system-wide', action='store_true',
+        parser.add_argument('-sw', '--system-wide', action='store_true',
```

Comment:
Argparse does not support the `-sw` as `-s -w`, anyway I understand it can be confusing. Let me just use the `--system-wide`.